src/App.js

```diff
@@ -11,11 +11,13 @@ const getPostsWithUserAndComments = (
   postList,
   userList,
   commentList
-) => postList.map(post => ({
-  ...post,
-  user: userList.find(user => user.id === post.userId),
-  comments: commentList.filter(comment => comment.postId === post.id),
-}));
+) => postList.map(
```

Comment:
I meant this
```
...
) => (
  postList.map(post => ({
    ...post,
    user: userList.find(user => user.id === post.userId),
    comments: commentList.filter(comment => comment.postId === post.id),
  }))
)
```

Reply:
Could you please draw an arrow on your screen, because I cannot understand, where exactly is the point, where the function is going from a new line?

Reply:
`...` represent destructuring and everything before it

Reply:
Like this?
![image](https://user-images.githubusercontent.com/32244670/64691478-6752a680-d49b-11e9-8442-491d4fb54a3a.png)

Reply:
`postList.map` is the first element of `getPostsWithUserAndComments` function, therefore parentheses should start before it as they wrap the function body

Reply:
Iryna, code is changed and added on github